etc/singularity.conf

```diff
@@ -141,3 +163,14 @@ container dir = /var/singularity/mnt
 # better option.
 #sessiondir prefix = /var/singularity/sessions/
 
+
+# SINGULARITY USER: [STRING]
+# DEFAULT: singularity
+# The protected image mode will restrict Singularity images to those owned by
+# the 'singularity user'; this option allows admins to change the username we
+# use as a test.
+#
+# If user namespaces are enabled, the singularity user setting is ignored and
```

Comment:
@bauerm97 -
Two options exist:
   1.  Simply keep the `allow user images = yes` default and allow user `mibauer` to use normal Unix permissions to share his images.
   2.  If `allow user images = no` is set for other reasons, you could add `mibauer` to the `singularity` group, set `protected image mode = user` and have him set those images to be owned by group `singularity`.
      * Note that this allows anyone on the host to access the image.

So (2) is close - but not exactly - to what you want.

I currently clear out the supplementary GID list when testing access from the `singularity` user.  I suppose we could keep these GIDS for the `singularity` user.  In that case, (2) would require that the image live in group-readable (not-world-readable) space and owned by the `singularity` group.